mongo-docker/start.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Start mongo
+/tmp/entrypoint.sh mongod&
```

Comment:
I'm curious how Docker handles this, with the `mongod` processes it put into the background.
* Do `mongod` logs get written to stdout/stderr and get picked up as Docker logs as when running in the foreground?
* Does the container stop when the `mongod` process dies?

Do `mongod` logs get logged to the console appropriately?  I know there can be issues when you put a process into the background.